source/dub/internal/utils.d

```diff
@@ -325,6 +325,83 @@ ubyte[] download(URL url, uint timeout = 8)
 	return download(url.toString(), timeout);
 }
 
+/**
+	Downloads a file from the specified URL with retry logic.
```

Comment:
I think the documentation could be improved here. Something like
```
Downloads a file from the specified URL with up to 3 trys on failure
Throws: `Exception` if the download failed or `HTTPStatusException` on "unrecoverable failures" such as 404 not found. Otherwise might throw anything else that `download` throws.
See_Also: download
```

Reply:
done